dbt/dbt_project.yml

```diff
@@ -28,6 +28,6 @@ models:
   athena:
     +materialized: view
     default:
-      +schema: dbt-test-default
```

Comment:
We no longer need to manually specify this prefix, because the `generate_schema_name` macro is doing it for us.